deployment-repository/src/main/java/org/jboss/as/repository/ContentRepository.java

```diff
@@ -72,7 +58,11 @@
      * Currently 5 minutes.
      */
     long OBSOLETE_CONTENT_TIMEOUT = getSecurityManager() == null ? getLong(Factory.UNSUPPORTED_PROPERTY, 300000L) : doPrivileged((PrivilegedAction<Long>) () -> getLong(Factory.UNSUPPORTED_PROPERTY, 300000L));
-
+    /**
+     * Timeout duiration to get a lock on a deployment for removing it or reading some content.
+     * Currently 1 minutes.
+     */
+    long LOCK_TIMEOUT = 60000L;
```

Comment:
This seems long, unless there's reason to expect that something preventing the lock would go away. We want to avoid failing due to a fairly long full gc or something but beyond that if it's not going to unlock the op shouldn't hang around waiting.